firestore/cloud-client/snippets.py

```diff
@@ -572,6 +572,16 @@ def order_where_limit():
     print(results)
 
 
+def order_limit_to_last():
+    db = firestore.Client()
+    # [START fs_order_by_name_limit_query]
+    cities_ref = db.collection("cities")
+    query = cities_ref.order_by("name").limit_to_last(2)
+    results = query.get()
+    # [END fs_order_by_name_limit_query]
+    print(results)
```

Comment:
Should the print be placed within the region tags?

Reply:
In other Firestore snippets prints are out of region tags, so it looks like no